src/sendmail/record.html

```diff
@@ -0,0 +1,43 @@
+<tr>
+    {% if record_number %}
+        <td>{{ record_number }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if record_type %}
+        <td>{{ record_type }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if project_name %}
+        <td>{{ project_name }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if address %}
+        <td>{{ address }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if expiration_date %}
+        <td>{{ expiration_date }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if short_notes %}
+        <td>{{ short_notes }}</td>
+    {% else %}
+        <td></td>
+    {% endif %}
+
+    {% if record_link %}
+        <td><a href="{{ record_link }}">View Record</a></td>
```

Comment:
If you take my suggestion about passing an object in to this template, this will actually become `record.record_details_link`

Reply:
Done!